### PR TITLE
demo/bitcoin-sentiment-analysis

### PR DESCRIPTION
**Pull request by bm64**
Added bitcoin sentiment analysis chart.

### CODE REVIEW

**Review by kamil-musialowski — APPROVED** [review]
Look and works really well 👏 
Does that mean, we should delete [this PR](https://github.com/highcharts/highcharts/pull/19511 )? @pawelfus

File: samples/stock/demo/flags-general/demo.js [hunk 1]
```diff
@@ -2,62 +2,65 @@
...
         series: [{
-            name: 'USD to EUR',
+            name: 'Bitcoin Price',
+            color: '#ffbf00',
             data: data,
-            id: 'dataseries'
+            id: 'dataseries',
+            tooltip: {
+                pointFormat: `<span style='color:{point.color}'>●</span>
+                    {series.name}: <b>{point.y:.2f}$</b><br/>`
```
Comment (DJTechnoo) [inline samples/stock/demo/flags-general/demo.js:30]: For the dollar sign, it should be to the left of the value

**Review by DJTechnoo — APPROVED** [review]
Looking good! Approving, but could you see my small comment?

**Comment by pawelfus** [comment]
> Look and works really well 👏 Does that mean, we should delete [this PR](https://github.com/highcharts/highcharts/pull/19511)? @pawelfus

Yes, we should. Nice catch!

File: samples/stock/demo/flags-general/demo.js [hunk 2]
```diff
@@ -2,62 +2,65 @@
...
-            shared: true
+            text: 'Bitcoin Historical Price and Halvings'
         },
 
-        yAxis: {
-            title: {
-                text: 'Exchange rate'
-            }
+        xAxis: {
+            overscroll: 31536000000 // 1 year
```
Comment (pawelfus) [inline samples/stock/demo/flags-general/demo.js:20]: It seems you added extra space for flag? However, this makes RS buttons a bit useless on initial extremes. What do you think about changing this to one month, then removing 1M button from RS and setting `shape: 'squarepin'` for flags? So the connector is rendered in the middle of flag, not on the left edge

**Review by pawelfus — APPROVED** [review]
Nice!

One suggestion: What do you think about adding simplified flags to the navigator (like 2nd, 3rd and 4th) and decreasing the initial range to 1y?

File: samples/stock/demo/flags-general/demo.js [hunk 3]
```diff
@@ -2,62 +2,65 @@
...
         series: [{
-            name: 'USD to EUR',
+            name: 'Bitcoin Price',
+            color: '#ffbf00',
             data: data,
-            id: 'dataseries'
+            id: 'dataseries',
+            tooltip: {
+                pointFormat: `<span style='color:{point.color}'>●</span>
+                    {series.name}: <b>$${'{point.y:.2f}'}</b><br/>`
```
Comment (pawelfus) [inline samples/stock/demo/flags-general/demo.js:30]: Please restore thousands separator ;)

**Review by pawelfus — CHANGES_REQUESTED** [review]

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**